src/test/java/com/rakovets/course/java/core/practice/jcf_set/TaxServiceTest.java

```diff
@@ -0,0 +1,174 @@
+package com.rakovets.course.java.core.practice.jcf_set;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.*;
+
+public class TaxServiceTest {
+    private final static Penalty SPEED_GDANSK = new Penalty("Speed", "Gdansk");
+    private final static Penalty SPEED_GDYNIA = new Penalty("Speed", "Gdynia");
+    private final static Penalty TECH_INSPECTION_GDYNIA = new Penalty("Technical Inspection", "Gdynia");
+    private final static Penalty TECH_INSPECTION_GDANSK = new Penalty("Technical Inspection", "Gdansk");
```

Comment:
Best way: duplicate those values in each tests

Reply:
fixed